source/resources/examples/typescript/aggregate/AddingCosignaturesAggregateComplete.ts

```diff
@@ -1,5 +1,8 @@
 // tslint:disable-next-line: max-line-length
-import { Account, AggregateTransaction, CosignatureSignedTransaction, CosignatureTransaction, Deadline, Mosaic, NamespaceId, NetworkCurrencyPublic, NetworkType, PlainMessage, PublicAccount, RepositoryFactoryHttp, TransactionMapping, TransferTransaction, UInt64 } from 'symbol-sdk';
+import { Account, AggregateTransaction, CosignatureSignedTransaction, CosignatureTransaction, Deadline, Mosaic, NamespaceId, NetworkCurrencies, NetworkType, PlainMessage, PublicAccount, RepositoryFactoryHttp, TransactionMapping, TransferTransaction, UInt64 } from 'symbol-sdk';
+
+// Retrieve from node's /network/properties
```

Comment:
from network/properties endpoint 
or getting from RepositoryFactory

Reply:
I added the comment.
I did not update the examples to retrieve the epoch adjustment from the repo factory because it complicates things way too much (toPromise() + await, and await cannot be used in the top level module...)

Reply:
somewhere it should be mentioned I guess. the epochAdjustment is not hardcoded anymore (used to be) since the adjustment value will be different between networks e.g. testnet  / mainnet